### ISSUE TITLE
Segment keywords

### ISSUE THREAD

**Issue by pcuci** [bettermtl/bettermtl.github.io#25]
Display event keywords on main landing page, quick "what's it about"

Replace title?

**Comment by David-Ro** [comment]
yes, replace title, after accepting it during setup. Let curators change it later, into a stream of linked keywords.

**Comment by pcuci** [comment]
Could be two fields: title and content keywords

The keywords could double as tags

**Comment by pcuci** [comment]
Hovering over keywords shows title
